tests/app/celery/v3/test_notification_tasks.py

```diff
@@ -22,7 +22,6 @@
 ############################################################################################
 
 # TODO - Make the Notification.template_id field nullable?  Have a default template?
```

Comment:
Can we remove this TODO now? Or is this part of some other feature yet to be built?